table.css

```diff
@@ -0,0 +1,27 @@
+/* exo 3 & 5 */
+table tbody tr:not(thead, :last-child, #hide):nth-child(odd) td, tbody tr:first-child {
```

Comment:
table tbody tr:nth-child(odd) td:not(:last-child) { 

Aurait été suffisant ici.
Pas besoin de désactiver les trame sur les #hide
La consigne demandait de laisser la dernière colonne non tramée et non la dernière ligne
Attention un tr ne pas pas être un thead en même temps, tr:not(thead) n'est pas bon dans la syntaxe